components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -10,57 +13,61 @@ type ProfileDetailCard = {
 function ProfileDetailCard({
   avatar,
   name,
+  link,
   joinDate,
   badges,
   currentFocus,
   nextGoal,
 }: ProfileDetailCard) {
+  let profile = "profile/" + link;
   return (
     <div className="bg-gray-900 text-sm w-60 p-6 h-full space-y-8 rounded-md ">
```

Comment:
why is this so dark lol 

is it because you're typically using dark mode?

Reply:
The cards are dark because I'm having trouble changing the text color from white. Met with Jason about it, he was also having trouble.